### DISCUSSION TITLE
"Mute Myself" indicator.

### DISCUSSION THREAD

**Discussion by terryt063** [jamulussoftware/jamulus#2589] [Ideas]
Much like on something like discord, would it be possible to put a bit like the speaker icon, but maybe a mic slashed through to indicate someone has muted themselves?

**Comment by dcorson-ticino-com** [comment]
Mute Myself shows the big red banner as it was seen that an icon was too easily overseen.
Even with the big banner we hear of cases when people are not aware that they have muted themselves.
Making that display smaller seems to me to be imprudent considering the number of cases that have been happening.

**Comment by terryt063** [comment]
Apologies, Was not intending that the current red banner is reduced as I agree, many people still don't get this right... I meant on top of this (to the user that has muted) an indicator to the room, that someone is on mute. So that the "left muted" issue may be resolved by someone in the server:- "You've muted yourself mate..." 

So similar to the indicator of "someone has muted you" on the slider, a similar mic looking icon, this person isn't sending audio...

**Reply by dcorson-ticino-com** [reply]
This has been discussed too, but it means sending the MuteMyself state to the others.
The present Jamulus protocol does not allow this, Mute Myself is a purely client function.
Maybe at some time in the future, it is in any case a feature that is being discussed.
